TodoList/TodoList.Api.Tests/Utils/ExtensionItemCompare.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NUnit.Framework.Constraints;
+
+namespace TodoList.Api.Tests.Utils
```

Comment:
I'm not a friend of "omnipotent" `Utils` folders in the solution. Please, use more specific names in C# projects - in this case, the folder should be named `Extensions` (by custom in .NET).